apps/docs/content/references/zsc.mdx

```diff
@@ -0,0 +1,249 @@
+
+### Inside zerops.yml
+
+You can use ZSC commands in your `zerops.yml` file.
+
+```yaml
+prepareCommands:
+  - zcli vpn up
+```
+
+```yaml
+buildCommands:
+  - zcli vpn up
+  - zcli vpn status
+```
+
+### VPN Connection
+
+You can connect to your project network with `zcli` command line tool and interact with your containers inside the project.
+
+```sh
+zcli vpn up
+```
+
+### Web Terminal
+
+Web Terminal allows you to interact with your service via remote web terminal.
+
+<p align="center">
+  <Image
+    lightImage="/img/gui/zsc-web-terminal.webp"
+    darkImage="/img/gui/zsc-web-terminal.webp"
+    alt="Zerops Setup Control Web Terminal"
+    caption="Web Terminal"
+    style={{ width: '100%', height: 'auto' }}
+  />
+</p>
+
+:::info
+The VPN connection is required to interact with your containers using ZSC commands.
+:::
+
+## Available Commands
+
+### backup-create and backup-current
+
+The `backup-create` command lets you create a backup of any specified stack in your project, while `backup-current` command creates a backup of the stack you're currently working in.
+
+```sh
+# Create backup of a specified stack
+zsc backup-create
+
+# Create backup of the current stack you're working in
+zsc backup-current
+```
+
+:::info
+Backups include all the data and configurations of the stack at the time of backup creation. These can be used for recovery or creating duplicate environments.
+:::
+
+### connectSharedStorage
+
+This command lets you connect a shared storage volume to your service for persistent data storage.
+
+```sh
+zsc connectSharedStorage
+```
+
+### crontab
+This command lets you manage scheduled tasks that are defined in your zerops.yml configuration.
+
+```sh
+zsc crontab
+```
+
+### execOnce
+
+:::info
+This command is particularly useful in high-availability setups to ensure commands (like database migrations) run only once, even when starting multiple containers.
+:::
+
+This command lets you execute a command exactly once across all containers in a service. Useful for database migrations in high-availability setups.
+
+```sh
+zsc execOnce "db-migration" "php artisan migrate"
+```
+
+### fail-me
+This command lets you deliberately fail the current container for testing purposes.
+
+```sh
+zsc fail-me
+```
+
+### install
+You can install additional base technologies in your runtime container that weren't specified in the initial configuration.
+
+:::info
+The `install` command allows you to add technologies that weren't specified in the initial base configuration.
+:::
+
+Example usage in `zerops.yml`:
+```yaml
+zerops:
+  - setup: nodejsapp
+    build:
+      os: ubuntu
+      base: 
+        - nodejs@22
+        - python@3.7
+    run:      
+      os: ubuntu
+      base: nodejs@22
+      prepareCommands:
+        - zsc install python@3.7
+```
+
+### noop
+An infinitely blocking command that does nothing and keeps the container alive so that it doesn't end because of an error.
```

Comment:
Add a note that it runs until it receives a SIGTERM signal.

Reply:
done ✅